CONTRIBUTING.md

```diff
@@ -0,0 +1,151 @@
+
+1. Do a quick search on GitHub to check if the issue has already been reported.
+2. Is it an issue with the Minetest *engine*? If not, report it [elsewhere](http://www.minetest.net/development/#reporting-issues).
+3. [Open an issue](https://github.com/minetest/minetest/issues/new) and describe the issue you are having - you could include:
+ - Error logs (check the bottom of the debug.txt file)
+ - Screenshots
+ - Ways you have tried to solve the issue, and whether they worked or not
+ - Your Minetest version and the content (subgames, mods or texture packs) you have installed
+ - Your platform (e.g. Windows 10 or Ubuntu 15.04 x64)
+
+After reporting you should aim to answer questions or clarifications as this helps pinpoint the cause of the issue (if you don't do this your issue may be closed after 1 month).
+
+## Requesting Features
+
+Feature requests are welcome but take a moment to think if your idea follows the [roadmap](https://forum.minetest.net/viewtopic.php?t=9177) in some way and fits the whole picture of the project. You should provide a clear explanation with as much detail as possible.
+
+## Translating
+
+Translations of Minetest are performed using a web interface called [Weblate](https://hosted.weblate.org/projects/minetest/minetest/).
+
+Current languages can be found in the po/ directory, create an issue if you'd like to translate into a language that isn't there.
+
+## Other ways of contributing
+
+### Creating Content
+
+You can create your own [mods](http://www.minetest.net/customize/#mods), [texture packs](http://www.minetest.net/customize/#texture-packs) or [subgames](http://www.minetest.net/customize/#subgames) and submit them to the [community](https://forum.minetest.net/).
+
+**Resources:**
+
+- [Mods](https://forum.minetest.net/viewforum.php?f=46) / [Texture packs](https://forum.minetest.net/viewforum.php?f=4) / [Subgames](https://forum.minetest.net/viewforum.php?f=48) forum
+- [Online Modding Tutorial Book](http://rubenwardy.com/minetest_modding_book/)
+- [Find Ideas](https://forum.minetest.net/viewtopic.php?f=9&t=2434)
+
+### Donate
+
+If you'd like to monetarily support Minetest, you can find donation methods on [the website](http://www.minetest.net/development/#donate).
+
+# Maintainer Notes
+
+*This is a concise version of the Rules & Guidelines on the [developer wiki](http://dev.minetest.net/Category:Rules_and_Guidelines).*
+
+These notes are for those who have push access Minetest (core developers / maintainers).
+
+## Subsystems
+
+Minetest is split into *subsystems* based on general parts of the code. Each subsystem is allocated to a certain core developer, who becomes the *subsystem maintainer*. They are responsible for handling pull requests and ensuring the subsystem works. They can give their responsibilities to someone else if they see fit.
+
+Subsystem maintainers can be proposed by anyone and they are not-so-officially majority voted.
+
+Developer            |Subsystem
```

Comment:
> Maybe link to the wiki, as these may change. (don't know what others think).

Good idea, I have linked to the Organisation page on the dev wiki (although that page needs to be updated).